src/syrupy/session.py

```diff
@@ -156,7 +156,7 @@ def _diff_snapshot_files(
         }
 
     def _count_snapshots(self, snapshot_files: SnapshotFiles) -> int:
-        return sum([len(snapshots) for _, snapshots in snapshot_files.items()])
+        return sum(map(len, snapshot_files.values()))
```

Comment:
I think most style guides prefer generators/comprehension here:
```
return sum(len(snaps) for snaps in snapshot_files.values())
```

Personally I prefer the more functional programming approach, i.e. chaining maps/reducers/etc, however I think it's not "pythonic".

Reply:
Originally had that but personally I prefer the more concise but still easy to read... but can change it for consistency 🤷‍♂

```suggestion
        return sum(len(snaps) for snaps in snapshot_files.values())
```